Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.9.0-slim-buster as base
+
+EXPOSE 5000
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+ENV PATH="${PATH}:/root/.poetry/bin"
+
+WORKDIR /app
+COPY . .
+
+RUN poetry install
```

Comment:
You probably want the `--no-dev` option.
And you actually need to add pytest, selenium etc as dev dependencies. I think you lost them when switching from pip to poetry.